features/networking/sdn.feature

```diff
@@ -300,3 +300,34 @@ Feature: SDN related networking scenarios
     Then the step should succeed
     And the output should contain "Hello OpenShift"
     """
+    
+  # @author anusaxen@redhat.com
+  # @case_id OCP-25787
+  @admin
+  Scenario: Don't write CNI configuration file until ovn-controller has done at least one iteration
+    Given the env is using "OVNKubernetes" networkType
+    And I select a random node's host
+    #Checking controller iteration 1
+    When I run command on the "<%= node.name %>" node's sdn pod:
+      | bash | -c | ovs-appctl -t ovn-controller connection-status |
+    Then the step should succeed
+    And the output should contain "connected"
+    #Checking controller iteration 2
+    When I run command on the "<%= node.name %>" node's sdn pod:
+      | bash | -c | ls -l /var/run/openvswitch/ |
+    Then the step should succeed
+    And evaluation of `@result[:response].match(/ovn-controller.\d*\.ctl/)[0]` is stored in the :controller_pid_file clipboard
```

Comment:
What will the stored ":controller_pid_file clipboard" be used for? Does not see other step to use this clipboard

Reply:
@weliang1 Iteration 3 is using it. This will confirm ,ovn controller has programmed the openvswitch flows and has connection with it.

Reply:
Sorry, I just saw it